.github/workflows/tag.yml

```diff
@@ -0,0 +1,27 @@
+name: Bump version tag
```

Comment:
@quesabe same as the comment above — let's move this workflow to `create-release-gh-action`.